vstgui/lib/platform/linux/cairopath.cpp

```diff
@@ -83,7 +83,7 @@ void GraphicsPath::addArc (const CRect& rect, double startAngle, double endAngle
 //------------------------------------------------------------------------
 void GraphicsPath::addEllipse (const CRect& rect)
 {
-#warning TODO: GraphicsPath::addEllipse
+	addArc(rect, 0, 360, true);
```

Comment:
Please use the coding style used in VSTGUI and add a space between function name and the opening bracket.